src/module.rs

```diff
@@ -1073,6 +1092,176 @@ impl DataLayout {
         }
         data_layout
     }
+
+    /// Port of `llvm::DataLayout::getTypeStoreSize`.
```

Comment:
Can we have a comment pointing to the upstream function this is a port of, similar to how you did for `getTypeSizeInBits`?